hashFuncs.go

```diff
@@ -0,0 +1,45 @@
+package haproxyconfigparser
```

Comment:
I think rename `hashFuncs.go` to `hashfuncs.go` is more natural.

Reply:
Sure will.